common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -885,3 +885,41 @@ def get_username(strategy, details, backend, user=None, *args, **kwargs):
     else:
         final_username = storage.user.get_username(user)
     return {'username': final_username}
+
+
+def set_learner_active_enterprise(user=None, backend=None, response=None, *args, **kwargs):
+    """
+    Check user active enterprise, if it matches to idp-enterprise then no action needed otherwise make the
```

Comment:
nit: Make 'active' a user's enterprise, if the currently 'active' enterprise in EnterpriseCustomerUser does not match the SAML idp-enterprise